source/threads.tex

```diff
@@ -7868,26 +7868,17 @@
 
 \indexlibrarymember{operator=}{unique_lock}%
 \begin{itemdecl}
-unique_lock& operator=(unique_lock&& u);
+unique_lock& operator=(unique_lock&& u) noexcept;
 \end{itemdecl}
 
 \begin{itemdescr}
 \pnum
 \effects
-If \tcode{owns} calls \tcode{pm->unlock()}.
-
-\pnum
-\ensures
-\tcode{pm == u_p.pm} and \tcode{owns == u_p.owns} (where \tcode{u_p} is the state of \tcode{u} just prior to this construction),  \tcode{u.pm == 0} and \tcode{u.owns == false}.
+Equivalent to: \tcode{unique_lock\{std::move(u)\}.swap(*this)}
```

Comment:
We had a request to change this editorially to use `()` instead of `{}` but I can submit a PR to do that after it's merged.

Reply:
Do feel free to attach an editorial commit to the PR right here, that seems helpful.

Reply:
Done in 8d517c60b0dfe875cc1b3b2961738941db889a0e

Reply:
Thanks!